backend/src/main/resources/db/migration/V1__hufu-ddl.sql

```diff
@@ -0,0 +1,35 @@
+/*
+ Navicat Premium Data Transfer
+
+ Source Server         : Hu-Fu
+ Source Server Type    : MySQL
+ Source Server Version : 50739 (5.7.39-0ubuntu0.18.04.2)
+ Source Host           : localhost:3306
+ Source Schema         : Hu-Fu
+
+ Target Server Type    : MySQL
+ Target Server Version : 50739 (5.7.39-0ubuntu0.18.04.2)
+ File Encoding         : 65001
+
+ Date: 16/10/2022 10:21:19
+*/
+
+SET NAMES utf8mb4;
+SET FOREIGN_KEY_CHECKS = 0;
+
+-- ----------------------------
+-- Table structure for sqlRecord
+-- ----------------------------
+DROP TABLE IF EXISTS `sqlRecord`;
+CREATE TABLE `sqlRecord` (
+  `id` bigint(20) NOT NULL,
+  `context` varchar(255) DEFAULT NULL,
+  `username` varchar(255) DEFAULT NULL,
+  `status` varchar(255) DEFAULT NULL,
+  `subTime` timestamp(3) NULL DEFAULT NULL,
+  `startTime` timestamp(3) NULL DEFAULT NULL,
```

Comment:
We use Camel-Case in Java. While in sql, we usually underline('_') to split different words in the column name.